### PR TITLE
new test branch

### PR DESCRIPTION
**Pull request by TeganAli**
# About

This is a post about...

# Pre-requisites

- [ ] The draft is complete and this post is ready to be reviewed.
- [ ] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

### CODE REVIEW

File: blog/2021-03/new-test-branch/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+author: tegan.ali@octopus.com
+visibility: public
+published: 2021-04-01
+metaImage: to-be-added-by-marketing
+bannerImage: to-be-added-by-marketing
+tags:
+  - DevOps
+---
+
+State the subject of the announcement as plainly and directly as you can.
```
Comment (wordlee) [inline blog/2021-03/new-test-branch/index.md:13]:
```suggestion
State the subject of the announcement as plainly and directly as you can. Don't waffle on.
```

File: blog/2021-03/new-test-branch/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+bannerImage: to-be-added-by-marketing
+tags:
+  - DevOps
+---
+
+State the subject of the announcement as plainly and directly as you can. Don't waffle on.
+
+## Body
+
+Body and Conclusion headings might be unnecessary for short posts.
```
Comment (wordlee) [inline blog/2021-03/new-test-branch/index.md:17]: Should conclusion be capitalized?

Comment (TeganAli) [reply blog/2021-03/new-test-branch/index.md:17]: I will change it, thanks

Comment (TeganAli) [reply blog/2021-03/new-test-branch/index.md:17]: I've removed capitalization

**Review by wordlee — APPROVED** [review]
Great test post! I made a couple of minor changes.

**Comment by wordlee** [comment]
This was just a test that we do not want to publishing.

**Pull request closed without merging**